plugins/azure/loadbalancer/lbPublicIp.js

```diff
@@ -0,0 +1,54 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Load Balancer Has Public IP',
+    category: 'Load Balancer',
+    domain: 'Availability',
+    description: 'Ensures that Azure Load Balancers has Public IP associated.',
+    more_info: 'A public IP associated with a load balancer serves as an Internet-facing frontend IP configuration.The frontend is used to access resources in the backend pool. The frontend IP can be used for members of the backend pool to egress to the Internet.',
```

Comment:
```suggestion
    more_info: 'A public IP associated with a load balancer serves as an Internet-facing frontend IP configuration.The frontend is used to access resources in the backend pool. The frontend IP can be used for members of the backend pool to egress to the Internet.',
```
maybe change it to pros of using a public load balancer